packages/inference/src/tasks/index.ts

```diff
@@ -40,3 +42,5 @@ export * from "./multimodal/visualQuestionAnswering";
 // Tabular tasks
 export * from "./tabular/tabularRegression";
 export * from "./tabular/tabularClassification";
+
+export const speechToText = automaticSpeechRecognition;
```

Comment:
(unrelated) aliasing `automaticSpeechRecognition` as `speechToText`

Reply:
did it exist before or that's new?

Reply:
(why not i guess, though we should try to push the "official" task names otherwise it's hard to standardize one way of doing things)

Reply:
> (why not i guess, though we should try to push the "official" task names otherwise it's hard to standardize one way of doing things)

yeah, good point